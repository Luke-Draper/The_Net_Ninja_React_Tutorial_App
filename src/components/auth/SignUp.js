import React, { Component } from "react";

class SignUp extends Component {
	state = {};
	render() {
		return (
			<div className="SignUp">
				<p>SignUp</p>
			</div>
		);
	}
}

export default SignUp;
