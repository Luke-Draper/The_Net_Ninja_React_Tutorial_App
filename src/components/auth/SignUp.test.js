import React from "react";
import SignUp from "./SignUp";
import { MemoryRouter } from "react-router-dom"; //Required when component uses a Route or Link
import renderer from "react-test-renderer";

test("SignUp renders without crashing", () => {
	const component = renderer.create(
		<MemoryRouter initialEntries={["/"]}>
			<SignUp />
		</MemoryRouter>
	);
	expect(component.toJSON()).toMatchSnapshot();
});
