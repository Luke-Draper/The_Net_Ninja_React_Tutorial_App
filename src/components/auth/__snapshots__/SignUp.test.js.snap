// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUp renders without crashing 1`] = `
<div
  className="SignUp"
>
  <p>
    SignUp
  </p>
</div>
`;
